content/docs/command-reference/status.md

```diff
@@ -13,82 +13,69 @@ usage: dvc status [-h] [-v] [-j <number>] [-q] [-c] [-r <name>] [-a] [-T]
+and `.dvc` files to compare the hash values of their <abbr>outputs</abbr>
+against the actual data files or directories in the workspace. The
+`--all-branches`, `--all-tags`, and `--all-commits` options enable checking data
+for multiple versions, not just the current branch.
+
+The `targets` given to this command (if any) limit what to check. It accepts
+paths to tracked files or directories (even if such paths are inside a
+[tracked directory](/doc/command-reference/add#tracking-directories)), `.dvc`
+files, or stage names (found in `dvc.yaml`).
+
+If no differences are detected, `dvc status` prints
+`Data and pipelines are up to date.` If differences are detected by
+`dvc status`, the command output indicates the changes. For each stage with
+differences, the changes in <abbr>dependencies</abbr> and/or
 <abbr>outputs</abbr> that differ are listed. For each item listed, either the
-file name or hash is shown, and additionally a status word is shown describing
-the changes (described below).
+file name or hash is shown, along with a _state description_, as detailed below:
 
-- _changed checksum_ means that the stage (in `dvc.yaml`) or `.dvc` file hash
-  has changed (e.g. someone manually edited the file).
+- _changed checksum_ means that the `.dvc` file hash has changed (e.g. someone
+  manually edited it).
 
-- _always changed_ means that this is a `.dvc` file with no dependencies (an
-  _orphan stage_ (see [`dvc add`](/doc/command-reference/add)) or that the stage
-  in `dvc.yaml` has the `always_changed: true` value set (see `--always-changed`
-  option in `dvc run`).
+- _always changed_ means that this is a `.dvc` file with no dependencies (see
+  `dvc add`) or that the stage in `dvc.yaml` has the `always_changed: true`
+  value set (see `--always-changed` option in `dvc run`).
 
 - _changed deps_ or _changed outs_ means that there are changes in dependencies
-  or outputs tracked by the stage in `dvc.yaml` or `.dvc` file. Depending on the
-  use case, commands like `dvc commit`, `dvc repro`, or `dvc run` can be used to
-  update the file. Possible states are:
+  or outputs tracked by the stage or `.dvc` file. Depending on the use case,
+  commands like `dvc commit`, `dvc repro`, or `dvc run` can be used to update
+  the file. Possible states are:
 
-  - _new_: An <abbr>output</abbr> is found in the workspace, but there is no
-    corresponding file hash saved in the `dvc.lock` or `.dvc` file yet.
+  - _new_: An <abbr>output</abbr> is found in the <abbr>workspace</abbr>, but
+    there is no corresponding file hash saved in the `dvc.lock` or `.dvc` file
+    yet.
   - _modified_: An output or <abbr>dependency</abbr> is found in the workspace,
     but the corresponding file hash in the `dvc.lock` or `.dvc` file is not up
     to date.
-  - _deleted_: The output or dependency is referenced in a `dvc.yaml` or `.dvc`
+  - _deleted_: The output or dependency is referenced in a `dvc.lock` or `.dvc`
```

Comment:
what happens if there is not `dvc.lock`?

Reply:
All data is reported as _modified_.

Reply:
👍